frontend/app/invoices/page.tsx

```
@@ -338,10 +327,22 @@ export default function InvoicesPage() {
               searchColumn={user.roles.administrator ? "billFrom" : undefined}
+              actions={
+                <>
+                  {user.roles.administrator ? (
+                    <Button variant="outline" size="small" asChild>
+                      <a href={export_company_invoices_path(company.id)}>
+                        <Download className="size-4" />
+                        Download CSV
+                      </a>
+                    </Button>
+                  ) : null}
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)